src/test/Assets/TestProjects/StartupHookWithInstanceMethod/StartupHookWithInstanceMethod.cs

```
@@ -0,0 +1,9 @@
+using System;
+
+internal class StartupHook
+{
+    public void Initialize()
+    {
+        Console.WriteLine("Hello from startup hook with instance method!");
```

This is a failure right? It should throw an exception if called.